hudi-common/src/main/java/org/apache/hudi/metadata/TimelineMergedTableMetadata.java

```diff
@@ -112,4 +116,15 @@ private void processNextRecord(HoodieRecord<? extends HoodieRecordPayload> hoodi
   public Option<HoodieRecord<HoodieMetadataPayload>> getRecordByKey(String key) {
     return Option.ofNullable((HoodieRecord) timelineMergedRecords.get(key));
   }
+
+  /**
+   * Returns the timestamp of the latest synced instant.
+   */
+  public Option<String> getSyncedInstantTime() {
```

Comment:
Could we call this something else? this basically returns the earliest unsynced instant time, right? Mixing `sync` terminology here is kind of throwing me off, trying to grok this.

Reply:
Renamed.